functions/starter-function/package.json

```diff
@@ -32,22 +32,22 @@
   },
   "homepage": "https://github.com/ThatConference/that-api-functions#readme",
   "devDependencies": {
-    "@babel/cli": "^7.10.5",
-    "@babel/core": "^7.11.4",
+    "@babel/cli": "^7.11.6",
```

Comment:
since we're not using babel, do you want to remove it, or was your intent to add it later on?

Reply:
This is in a different function. I tagged on security updates and package bumps into this pr for the other functions.